tensorboard/webapp/metrics/store/metrics_store_internal_utils_test.ts

```diff
@@ -422,6 +433,20 @@ describe('metrics store utils', () => {
         card1: createCardMetadata(),
         [pinnedCardId]: createCardMetadata(),
       });
+      expect(cardStateMap).toEqual({
+        card1: {
+          timeSelection: {
+            start: {step: 5},
+            end: {step: 7},
+          },
+        },
+        ['{"baseCardId":"card1"}']: {
```

Comment:
When a card is pinned, the new id is generated using `getPinnedCardId`. I'm opting to specify the returned value here, but it might read better of this line used the function instead
```suggestion
        [getPinnedCardId('card1')]: {
```